src/client-common/client-telemetry-helper.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why this module is defined under client-common? If it is shared between clients like sim-host and/or app-host, then this can be defined under src/modules.